dtde/spark/cli/init.py

```diff
@@ -0,0 +1,20 @@
+import os 
+import argparse
+import typing
+from distutils.dir_util import copy_tree
+import dtde.constants as constants
+
+
+def setup_parser(parser: argparse.ArgumentParser):
+    pass
+
+def execute(args: typing.NamedTuple):
+    config_src_path = constants.INIT_DIRECTORY_SOURCE
+    config_dest_path = constants.INIT_DIRECTORY_DEST
+
+    copy_tree(config_src_path, config_dest_path, update=1)
```

Comment:
What happens if the source path already exists?

Reply:
Might be misunderstanding the question, but config_src_path exists when the repo is pulled. It is the config/ directory that is included in the repo.

config_dest_path is the .thunderbolt directory we are creating in the cwd. If that directory already exists and contains files that are in config_src_path, the files are left untouched. So running `azb spark init` twice in the same directory will not overwrite existing files.